src/kernel/arch/aarch64/interrupts.zig

```diff
@@ -5,8 +5,8 @@ pub extern var exception_table: *usize;
 
 comptime {
     asm (
-        \\.globl exception_table
-        \\exception_table:
+        \\.globl exception_table2
+        \\exception_table2:
```

Comment:
What's the purpose of this? I'm assuming that it's just left over from your testing.

Reply:
This was just to show what the current table is, to compare to the new table.